docs/cloud.md

```diff
@@ -0,0 +1,69 @@
+# Deploying Antrea on a cloud provider
+
+Antrea may run in networkPolicyOnly mode in EKS and AKS.
+
+## Pre-requisites
+1. to run EKS cluster, install and configure aws cli, see
+   https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-install.html, and
+   https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-configure.html
+2. to run AKS cluster, install and configure azure cli,
+   https://docs.microsoft.com/en-us/cli/azure/install-azure-cli?view=azure-cli-latest
+3. install terraform
+   see https://learn.hashicorp.com/terraform/getting-started/install.html
+
+## Create an AKS cluster via terraform
+create or obtain a azure service principal. And set the following environment variables from the service principal. For instance in .bashrc
```

Comment:
just like for EKS, I'd like to see some more detailed instructions. Here is what I did with the Azure portal:
1) Create a service principal by following the instructions in https://docs.microsoft.com/en-us/azure/active-directory/develop/howto-create-service-principal-portal#create-an-azure-active-directory-application. Copy the "Application (client) ID" value and use it to set `TF_VAR_aks_client_id`.
2) Create a secret for the application, copy the value and use it to set `TF_VAR_aks_client_secret`.
3) Create a resource group in the region of your choice and set `TF_VAR_aks_resource_group_name` to the name of the resource group.

Reply:
incorporated link into the doc.